src/pages/Registration.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    Checkbox,
+    FormControl,
+    FormControlLabel,
+    Grid,
+    Link,
+    TextField,
+    Typography,
+} from '@mui/material';
+import Topbar from 'components/header/Topbar';
+import React, { useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+interface RegistrationRequest {
+    firstName: string;
+    lastName: string;
+    email: string;
+    company: string;
+    useCase: string;
+    acknowledgedDocuments: boolean;
+}
+
+interface Errors {
+    firstName: boolean;
+    lastName: boolean;
+    email: boolean;
+    company: boolean;
+    useCase: boolean;
+    acknowledgedDocuments: boolean;
+}
+
+const Registration = () => {
```

Comment:
We need to make a call soon if we want to hand code forms like below or just handle them with JSON Schema (.../capture/create/DetailsForm.tsx as an example).

Reply:
Agreed. I thought this page served as a perfect opportunity to experiment with and demonstrate the implementation of a form without the support of JSON Schema.

Obviously, this is a very simple form. That said, it does highlight some of the complexity of this approach and functions as a concreate code sample for comparison.

Consider the topic added to list of UI discussion items.